### DISCUSSION TITLE
Domain Configuration

### DISCUSSION THREAD

**Discussion by hatim85f** [vercel/community#4333] [Help] [answered]
### Summary

I did add the DNS as shown to my goDaddy domain and the website is being directed but with the same Vercel domain not min in GoDaddy

### Example

www.stap-crm.com

### Steps to Reproduce

I did add the DNS as shown to my goDaddy domain and the website is being directed but with the same Vercel domain not min in GoDaddy

**Comment by amyegan** [comment, marked as answer]
Hi @hatim85f! The `DNS_PROBE_FINISHED_NXDOMAIN` error means that the domain could not be found.

I believe this domain is misconfigured. When I check the DNS records for `www.stap-crm.com`, I can see that there is a [CNAME record for www.stap-crm.com](https://www.whatsmydns.net/#CNAME/www.stap-crm.com), but [no A record for stap-crm.com](https://www.whatsmydns.net/#A/stap-crm.com) or [for www.stap-crm.com](https://www.whatsmydns.net/#A/www.stap-crm.com). 

For help adding and configuring a custom domain for project, please go to https://vercel.com/docs/projects/domains/add-a-domain

**Comment by hatim85f** [comment]
Hi @amyegan . Thank you for your response. I did reconfigure the DNS details and now it is working as expected
